lib/cisco_node_utils/snmpnotification.rb

```diff
@@ -0,0 +1,60 @@
+# December 2015
+#
+# Copyright (c) 2014-2015 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require_relative 'node_util'
+
+module Cisco
+  # SnmpNotification - node utility class for SNMP notification management
+  class SnmpNotification < NodeUtil
+    @notifications = nil
+
+    def initialize(name)
+      fail TypeError unless name.is_a?(String)
+      @name = name
+    end
+
+    def self.notifications
```

Comment:
I don't see any tests for this method.

1) Collect empty
2) Collect one
3) Collect many